packages/ui/src/machines/authenticator/signUp.ts

```diff
@@ -114,7 +114,7 @@ export function createSignUpMachine({ services }: SignUpMachineOptions) {
                       {
                         cond: 'shouldSkipConfirm',
                         target: 'skipConfirm',
-                        actions: ['setUser'],
+                        actions: ['setUser', 'setCredentials'],
```

Comment:
We need to set the credentials and the user so the `authAttributes` are set. Otherwise the `Auth.signIn` will fail later on, since the credentials will be undefined.